.prettierrc

```diff
@@ -0,0 +1,11 @@
+{
+  "printWidth": 120,
+  "tabWidth": 2,
+  "useTabs": false,
+  "semi": true,
+  "trailingComma": "none",
```

Comment:
I'd like to revisit this. There are a lot of situations where I would like a trailing comma, it makes some diffs much easier to read

Reply:
Agreed. I've disabled it since it did feel really weird. Have to read up on this more